eoscostarica/guide.eoscostarica.io
# Adding to developer guidelines ipfs investigation

### doc(frontend): adding to developer guidelines ipfs investigation

### What does this PR do?

- resolves #443 

### Steps to test
1. Go to developer guidelines
1. Click on IPFS (Interplanetary File System) in the sidebar
1. See the new doc.

#### CheckList
